styleguide-app/Examples/Page.elm

```diff
@@ -31,7 +31,13 @@ example =
     { name = "Nri.Ui.Page.V3"
     , categories = List.singleton Pages
     , state = ()
-    , update = \_ state -> ( state, Cmd.none )
+    , update =
+        \msg model ->
+            let
+                _ =
+                    Debug.log "Clicked: " msg
+            in
+            ( model, Cmd.none )
```

Comment:
🎨 do we want to keep this debug in?

Reply:
Yes, I think so -- many of the component examples currently use a debug.log to show that they do something 🤷‍♀